docker-compose.override.yml

```diff
@@ -4,13 +4,7 @@ services:
     command: python src/manage.py runserver 0.0.0.0:8000
     volumes:
       - .:/code
-      - /code/src/sous-chef
-      - /code/src/sous-chef/static
     expose:
       - "8000"
     ports:
```

Comment:
You are right on this too! :+1: 
I have tested and it works just fine without the `expose: - "8000"` but I don't really understand how the **Nginx container** can redirect the request to the **Web container** through the port **"8000"** if the port is not exposed. This work just fine without the `expose: - "8000"` so I will remove that.

We will be able to access the application using "http://0.0.0.0", "http://0.0.0.0:8000", "http://127.0.0.1" or "http://127.0.0.1:8000" in development mode because the containers bind the ports `Nginx:80 -> 80/tcp`,  `Web:8000 -> 8000/tcp` to the host. 

When we use "http://127.0.0.1" and  "http://0.0.0.0" the **Nginx container** will serve the static files and redirect other requests to the **Web container** that use the **Django development server**.

When we use "http://0.0.0.0:8000" or "http://127.0.0.1:8000" the **Web container** will handle all the requests running the **Django development server**.

In production mode, we can only access using "http://0.0.0.0" or "http://127.0.0.1" because only the `Nginx:80 -> 80/tcp` port is bind and the **Web container** use **Gunicorn** instead of the **Django development server**.